lib/corefoundation/version.rb

```diff
@@ -1,4 +1,4 @@
 module CF
   # The current version string
-  VERSION = "0.3.14"
+  VERSION = "0.4.1"
```

Comment:
This is updated through automation and does not need manual changes.

Reply:
@rishichawda file removed

Reply:
@alezummo tagged the PR for a minor version bump, as it seems prudent